I do need help with the failing tests:

``` 
java.lang.AssertionError: Error initializing database: Flyway migration error occurred: Error while executing afterMigrate callback: java.lang.NullPointerException
	at org.junit.Assert.fail(Assert.java:89)
	at org.dspace.AbstractUnitTest.initDatabase(AbstractUnitTest.java:80)
```

Even more clear when I try to run it locally:

```
[ERROR] Failures: 
[ERROR]   OrcidBulkPushIT>AbstractIntegrationTestWithDatabase.initDatabase:94 Error initializing database: Flyway migration error occurred: Error while executing afterMigrate callback: java.lang.NullPointerException: Cannot invoke "org.dspace.core.DBConnection.getSession()" because the return value of "org.dspace.core.Context.getDBConnection()" is null
```

Is there a configuration or anything missing that causes the DB Connection to be null?

--- a/dspace-api/src/main/java/org/dspace/orcid/OrcidHistory.java
+++ b/dspace-api/src/main/java/org/dspace/orcid/OrcidHistory.java
@@ -79,6 +79,8 @@ public class OrcidHistory implements ReloadableEntity<Integer> {
     /**
      * A description of the synchronized resource.
      */
+    @Lob
+    @Type(type = "org.dspace.storage.rdbms.hibernate.DatabaseAwareLobType")
     @Column(name = "description")
     private String description;
 

--- a/dspace-api/src/main/java/org/dspace/orcid/OrcidQueue.java
+++ b/dspace-api/src/main/java/org/dspace/orcid/OrcidQueue.java
@@ -64,6 +64,8 @@ public class OrcidQueue implements ReloadableEntity<Integer> {
     /**
      * A description of the resource to be synchronized.
      */
+    @Lob
+    @Type(type = "org.dspace.storage.rdbms.hibernate.DatabaseAwareLobType")
     @Column(name = "description")
     private String description;
 

--- a/dspace-api/src/test/java/org/dspace/orcid/script/OrcidBulkPushIT.java
+++ b/dspace-api/src/test/java/org/dspace/orcid/script/OrcidBulkPushIT.java
@@ -216,6 +216,62 @@ public class OrcidBulkPushIT extends AbstractIntegrationTestWithDatabase {
     }
 
     @Test
+    public void testWithVeryLongTitleQueueRecords() throws Exception {
+        Item firstProfileItem = createProfileItemItem("0000-1111-2222-3333", eperson, BATCH);
+        Item firstEntity = createPublication("Publication with a very very very very very very very very very " +
+                "very very very very very very very very very very very very very very very very very very very very " +
+                "very very very very very very very very very very very very very very very very very even " +
+                "extremely long title");
+
+        when(orcidClientMock.push(any(), eq("0000-1111-2222-3333"), any()))
+                .thenReturn(createdResponse("12345"));
+
+        when(orcidClientMock.update(any(), eq("0000-1111-2222-3333"), any(), eq("98765")))
+                .thenReturn(updatedResponse("98765"));
+
+        when(orcidClientMock.deleteByPutCode(
+                any(),
+                eq("0000-1111-2222-3333"),
+                eq("22222"),
+                eq("/work"))
+        ).thenReturn(deletedResponse());
+
+        createOrcidQueue(context, firstProfileItem, firstEntity);
+        createOrcidQueue(context, firstProfileItem, "Description", "Publication", "22222");
+
+        context.commit();
+
+        TestDSpaceRunnableHandler handler = runBulkSynchronization(false);
+
+        String firstProfileItemId = firstProfileItem.getID().toString();
+
+        assertThat(handler.getInfoMessages(), hasSize(5));
+        assertThat(handler.getInfoMessages(), containsInAnyOrder(
+                "Found 2 queue records to synchronize with ORCID",
+                "Addition of Publication for profile with ID: " + firstProfileItemId,
+                "History record created with status 201. The operation was completed successfully",
+                "Deletion of Publication for profile with ID: " + firstProfileItemId + " by put code 22222",
+                "History record created with status 204. The operation was completed successfully"));
+
+        assertThat(handler.getErrorMessages(), empty());
+        assertThat(handler.getWarningMessages(), empty());
+
+        verify(orcidClientMock).push(any(), eq("0000-1111-2222-3333"), any());
+        verify(orcidClientMock).deleteByPutCode(
+                any(),
+                eq("0000-1111-2222-3333"),
+                eq("22222"),
+                eq("/work"));
+
+        verifyNoMoreInteractions(orcidClientMock);
+
+        List<OrcidHistory> historyRecords = orcidHistoryService.findAll(context);
+        assertThat(historyRecords, hasSize(2));
+        assertThat(historyRecords, hasItem(matches(history(firstProfileItem, firstEntity, 201, INSERT))));
+        assertThat(historyRecords, hasItem(matches(history(firstProfileItem, 204, DELETE))));
+    }
+
+    @Test
     public void testWithOneValidationError() throws Exception {
 
         Item firstProfileItem = createProfileItemItem("0000-1111-2222-3333", eperson, BATCH);

